package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "identity-wallet-workspace",
```

Comment:
We can leave out properties such as `"name"`, `"license"` and `"private"` since we're never going to publish the package anyway.

Reply:
Fields removed.